nginx.conf

```diff
@@ -1,15 +1,29 @@
-server { 
- listen 9001 ssl;
- error_log stderr;
- access_log /dev/stdout;
-
- server_name localhost;
- ssl_certificate /var/serving-cert/tls.crt;
- ssl_certificate_key /var/serving-cert/tls.key;
-
- location / {
-  alias /opt/clowder-plugin/;
- }
+worker_processes  1;
+
+events {
+    worker_connections  1024;
 }
 
-# ~ ^\/apps\/([^\/]+)
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+
+    keepalive_timeout  65;
```

Comment:
mind to mention why are we adding these ?

Reply:
because we are taking away the defaults file - I thought it would be good to make it explicit

Reply:
Sorry, it isn't clear to me as these values differ from Nginx's default values. `keepalive_timeout` for example, is set to 65, but [the default value is 75](http://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_timeout). 
I just wanted to understand what motivated this fine tuning